src/administration/users.md

```diff
@@ -2,59 +2,38 @@
+* **Environment Reader** - An environment reader can only view this environment.
 
 > **Important!**
-> After a new user is added to an environment, it needs to be rebuilt. Rebuilds
-> are triggered when you push a new commit to the environment in question.
-> To be able to rebuild without new code changes you can issue the command
-> `git commit --allow-empty -m'rebuild' && git push`
-> to create an empty commit and "force" rebuilding the environment.
 >
-> When the environment rebuild is complete, allow a minute for the routes to
-> update fully and for the new user to be able to use SSH access.
+> After a new user is added to an environment, it needs to be rebuilt. Rebuilds are triggered when you push a new commit to the environment in question.  To be able to rebuild without new code changes you can issue the command `git commit --allow-empty -m'rebuild' && git push` to create an empty commit and "force" rebuilding the environment.
+>
+> When the environment rebuild is complete, allow a minute for the routes to update fully and for the new user to be able to use SSH access.
 
 ------------------------------------------------------------------------
 
-When a development team works on a project, the team leader can be the
-project administrator and decide which roles to give his team members.
-One team member can contribute to one environment, another member can
-administer a different environment and the customer can be a reader of
-the `master` environment.
+When a development team works on a project, the team leader can be the project administrator and decide which roles to give his team members.  One team member can contribute to one environment, another member can administer a different environment and the customer can be a reader of the `master` environment.
 
-If you want your users to be able to see everything (Reader), but only
-commit to a specific branch, change their permission level on that
-environment to "Contributor".
+If you want your users to be able to see everything (Reader), but only commit to a specific branch, change their permission level on that environment to "Contributor".
 
 > **SSH Access Control**
-> An environment contributor can push code to the environment and has
-> SSH access to the environment. You can change this by [specifying
-> user types](/configuration/app/access.md) with ssh access.
+>
+> An environment contributor can push code to the environment and has SSH access to the environment. You can change this by [specifying user types](/configuration/app/access.md) with ssh access.
 
 > **note**
-> The project owner - the person licensed to use Platform.sh - doesn't have
-> special powers. A project owner usually has a project administrator role.
+>
+> The project owner - the person licensed to use Platform.sh - doesn't havespecial powers. A project owner usually has a project administrator role.
```

Comment:
havespecial

Reply:
I KNEW that would happen somewhere. Fixed.